pennylane/kernels/cost_functions.py

```diff
@@ -15,7 +15,7 @@
 This file contains functionalities for kernel related costs.
 See `here <https://www.doi.org/10.1007/s10462-012-9369-4>`_ for a review.
 """
-from pennylane import numpy as np
+from pennylane import numpy as pnp  # using vanilla numpy here causes a circular import
```

Comment:
? how? its an external package?

Reply:
if I use vanilla numpy here, something tries to import pennylane numpy later and that causes an issue? I was going to get back to this later by doing something in the init file. lemme push a fix now